docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+8.  Add a **Title** for the event and a **Description** (optional).
+
+9.  Select the Segments you’d like to add to or remove the Contact from.
+
+10. Click **Add**.
+
+11. Click **Save & Close**.
+
+This accomplishes the same thing as creating a Dynamic Segment with a filter for Points, the operator ``greater than (or equal to)``, and the minimum number of Points. The difference is if you only want to add Contacts to a Segment who reach the point value after creating this trigger to a Segment, you can.
+
+.. vale off
+
+CSV upload
+===========
+
+.. vale on
+
+If you have a list outside of Mautic, saved as a UTF-8 encoded CSV file, you can upload the list directly into a Mautic Segment.
+
+1. Click **Contact**.
+
+2. Click the menu in the upper-right hand corner and select **Import**.
+
+3. Ensure the file is in a UTF-8 CSV format. Select your file then click **Upload**.
+
+4. At the top of the next screen, select the Segment you want to add the Contacts to.
+
+5. Map the appropriate fields from the file. Remember to always map the unique identifier, so you don’t create duplicates.
+
+6. Click **Import**.
+
+If your file is larger than 1,000 rows, the system changes screens and informs you once the file has uploaded. After the upload is complete, click the Segments section of the platform to see the added Contacts.
+
+.. vale off
+
+Dynamic Segments
+****************
+
+.. vale on
+
+.. image:: images/filtering.png
+    :alt: Screenshot showing Segment filters.
+
+Mautic moves Contacts into and out of dynamic Segments based on the filters applied to the Segment. As the data associated with the Contact updates, including Company associations and behaviors, Mautic updates Segment membership.
+
+Segment filters
+===============
+
+.. image:: images/filter-list.png
+    :width: 400
+    :alt: Screenshot showing Segment Filters List.
+
+1. Create a new Segment by clicking the **+New** button.
+
+2. In the **Details** tab, add a **Name**, **Public name** and **Description** to your Segment.
+
+3. Click the **Filters** tab to add filters.
+
+4. Click the **Choose one**… menu and search for the field you’d like to Segment by. Listed below are three types of fields:
+      
+  * Contact fields
+  
+    * Set Fields to **Available for Segments = Yes** in your Custom Field manager to display here.
+  * Contact behavior and actions
+  * Primary Company fields
+    * Set Fields to **Available for Segments = Yes** in your Custom Field manager to appear here.
+    * Contacts associates with multiple Companies, but Mautic adds them to Segments based on fields for the primary Company.
+
+5. Add more filters, using the **And** and **Or** operators. An **Or** operator creates a new group of filters which can include And operators.
+
+6. Click **Save and close**.
+
+.. note:: 
+
+    The following filters rebuild immediately in real time, while other filters rebuild with the nightly refresh:
```

Comment:
This is Acquia-specific so we should instead be replacing this with a note that segments are rebuilt according to how frequently you fire your cron jobs, and pointing people at the cron jobs docs about building segments.